e2e/Cargo.toml

```diff
@@ -17,7 +17,7 @@
 #
 [package]
 name = "e2e"
-version = "0.2.0"
+version = "0.0.0"
```

Comment:
Change to `0.0.0` because the version of e2e isn't important.

Reply:
0.0.0 means using the latest version?

Reply:
Should mean there is no version, since it doesn't have to be published to `crates.io`, it doesn't matter.

Reply:
I mean does this guarantee it would use the latest build version?

Reply:
This line insures to use the latest version.
<https://github.com/apache/skywalking-rust/blob/d14a0bd4c781671e907c3cb796f20ef23d6ed114/e2e/Cargo.toml#L27>